File: 1.Two Sum/two_sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int, int> dig;
        for(int i = 0; i < nums.size(); i++){
            if (dig.count(target - nums[i])) {
                return {i, dig[target - nums[i]]};
            }
            dig[nums[i]] = i;
        }
        return {};  
    }
};

File: 1.Two Sum/two_sum.py
class Solution(object):
    def twoSum(self, nums, target):
        numMap = {}
        ans = []

        for i in range(len(nums)):
            if target - nums[i] in numMap:
                return [numMap[target-nums[i]], i]
            else:
                numMap[nums[i]] = i
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        